config/initializers/monkey_patches.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-# Copyright 2015-2017, the Linux Foundation, IDA, and the
-# CII Best Practices badge contributors
-# SPDX-License-Identifier: MIT
-
-Dir[Rails.root.join('lib', 'ext', '**', '*.rb')].each { |file| require file }
```

Comment:
I just figured we could add it back if we wanted to do any monkey patches in the future. Given our use of [concerns](https://github.com/coreinfrastructure/best-practices-badge/tree/master/app/models/concerns), I didn't think that was likely.